(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
(IndexError('tuple index out of range',),)
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
Traceback (most recent call last):
  File "/home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/appointments/views.py", line 23, in get
    doctors = User.objects.filter(role=ROLE[10][0])
IndexError: tuple index out of range
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
({'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': IndexError('tuple index out of range',)},)
({'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': IndexError('tuple index out of range',)},)
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'error while Invalid verification link', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'uohfd', 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': "it's message", 'class_name': 'EmailValidate', 'request_method': 'GET', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'message': 'it; messaaggeeee', 'class_name': 'specialities', 'errors': IndexError('tuple index out of range',)}
{'message': 'it; messaaggeeee', 'class_name': 'specialities', 'errors': IndexError('tuple index out of range',)}
{'message': 'it; messaaggeeee', 'class_name': 'specialities', 'errors': IndexError('tuple index out of range',)}
tuple index out of range
tuple index out of range
tuple index out of range
(IndexError('tuple index out of range',),)
({'message': 'dfdbj', 'class_name': 'fdosjf', 'method_name': 'get', 'errors': IndexError('tuple index out of range',)},)
(<appointments.views.SpecialitiesView object at 0x7f0505e709e8>, IndexError('tuple index out of range',))
{'class_name': 'AppLogger', 'errors': (<appointments.views.SpecialitiesView object at 0x7fdb9c08a470>, IndexError('tuple index out of range',))}
{'class_name': 'AppLogger', 'errors': (<appointments.views.SpecialitiesView object at 0x7fa34ea8c1d0>, IndexError('tuple index out of range',))}
{'class_name': 'SpecialitiesView', 'errors': (IndexError('tuple index out of range',),)}
{'class_name': 'SpecialitiesView', 'errors': (IndexError('tuple index out of range',),)}
{'class_name': 'SpecialitiesView', 'method_name': 'get', 'errors': (IndexError('tuple index out of range',),)}
{'class_name': 'SpecialitiesView', 'method_name': 'get', 'errors': ()}
{'class_name': 'SpecialitiesView', 'method_name': 'get', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'get', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'get', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'get', 'errors': NameError("name 'doctor' is not defined",)}
{'class_name': 'AddDoctorView', 'method_name': 'get', 'errors': NameError("name 'Truee' is not defined",)}
{'class_name': 'AddDoctorView', 'method_name': 'get', 'errors': EncodeError(TypeError('Object of type AddDoctorView is not JSON serializable',),)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'AddDoctorView', 'method_name': 'POST', 'errors': EncodeError(TypeError('Object of type AddDoctorView is not JSON serializable',),)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
{'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 07:53:49,945 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 07:55:04,426 - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
ERROR:django.request:Internal Server Error: /appointments/specialities/
Traceback (most recent call last):
  File "/home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/appointments/views.py", line 23, in get
    doctors = User.objects.filter(role=ROLE[10][0])
IndexError: tuple index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/er/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/er/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/er/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/er/.local/lib/python3.6/site-packages/django/contrib/auth/mixins.py", line 71, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/er/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/appointments/views.py", line 29, in get
    AppLogger().log_error(var)
  File "/home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py", line 57, in __init__
    logging.basicConfig(filename=LOGS_PATH, fmt=LOG_FORMAT)
  File "/usr/lib/python3.6/logging/__init__.py", line 1829, in basicConfig
    raise ValueError('Unrecognised argument(s): %s' % keys)
ValueError: Unrecognised argument(s): fmt
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:/home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
2022-03-09 07:55:36,854 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 07:58:01,327 - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
2022-03-09 07:58:22,085 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:21:46,458 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:22:25,815 - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
2022-03-09 08:22:32,413 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:26:45,477 - root - ERROR - {'class_name': 'AddDoctorView', 'method_name': 'POST', 'errors': EncodeError(TypeError('Object of type AddDoctorView is not JSON serializable',),)}
2022-03-09 08:28:20,891 - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
2022-03-09 08:29:52,322 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:30:02,587 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:43:38,152 - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
2022-03-09 08:43:39,310 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:46:57,318 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:47:01,739 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 08:47:10,490 - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
2022-03-09 09:30:49,832 - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
03/09/2022 09:30:58 AM - root - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 10:03:04 AM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 10:03:42 AM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 10:05:31 AM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 10:07:55 AM - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/settings.py changed, reloading.
03/09/2022 03:38:06 PM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 03:38:46 PM - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/settings.py changed, reloading.
03/09/2022 03:47:39 PM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 03:47:53 PM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 03:48:50 PM - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
03/09/2022 04:02:17 PM - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
03/09/2022 04:12:44 PM - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
03/09/2022 04:29:33 PM - django.utils.autoreload - INFO - /home/er/Desktop/Appointment-Booking-System/AppointmentBookingSystem/AppointmentBookingSystem/logger.py changed, reloading.
03/09/2022 04:30:30 PM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
03/09/2022 04:30:34 PM - AppointmentBookingSystem.logger - ERROR - {'class_name': 'SpecialitiesView', 'method_name': 'GET', 'errors': IndexError('tuple index out of range',)}
